lib/cleaned_output.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'colorize'
+
+class CleanedOutput
+  def initialize(output:, morph_api_key: ENV['MORPH_API_KEY'])
+    @output = output
+    @morph_api_key = morph_api_key
+  end
+
+  def to_s
+    (cleaned_output[-64_000..-1] || cleaned_output).uncolorize
+  end
+
+  private
+
+  attr_reader :output, :morph_api_key
+
+  def cleaned_output
```

Comment:
this is overloading what 'cleaned' means in a slightly odd way, where `cleaned_output` here is something different to the `CleanedOutput` of the class. This is really just the original output with redactions applied.